dm/master/election.go

```diff
@@ -152,4 +153,5 @@ func (s *Server) retireLeader() {
 	s.leader = ""
 	s.closeLeaderClient()
 	s.Unlock()
+	metrics.RemoveDDLPendingInMetrics()
```

Comment:
this metric will reset to 0 when the leader is changed, may puzzles users  🤔

Reply:
I'm not sure how we supposed to use HA metrics. I think we could collect all dm-master's counter and add them, so non-leader should output zero.

Maybe we could forward metric requests to leader 😂 ?

Reply:
how about wait Xuecheng give some suggestion

Reply:
Maybe PD needs to solve the same problem, can we refer to it?

Reply:
PD use a reversed proxy (not understand) to forward every http request to leader, so this PR should be updated (but after I figure it out)